tests/integration/commands/deploy/deploy.test.mjs

```diff
@@ -712,85 +736,78 @@ if (process.env.NETLIFY_TEST_DISABLE_LIVE !== 'true') {
-            content: JSON.stringify({
-              functions: [
-                {
-                  ...bundledFunctionData,
-                  path: join(builder.directory, '.netlify', 'functions', 'bundled-function-1.zip'),
-                },
-              ],
-              timestamp: Date.now() - age,
-              version: 1,
-            }),
-          })
-          .withContentFile({
-            path: 'out/index.html',
-            content: 'Hello world',
-          })
-          .withFunction({
-            path: 'bundled-function-1.js',
-            handler: async () => ({
-              statusCode: 200,
-              body: 'Bundled at deployment',
-            }),
-          })
-          .buildAsync()
-
-        const { deploy_url: deployUrl } = await callCli(
-          ['deploy', '--json'],
-          {
-            cwd: builder.directory,
-            env: { NETLIFY_SITE_ID: t.context.siteId },
+    await withSiteBuilder('site-with-functions-manifest-3', async (builder) => {
+      await builder
+        .withNetlifyToml({
+          config: {
+            build: { publish: 'out' },
+            functions: { directory: 'functions' },
           },
-          true,
-        )
+        })
+        .withCopiedFile({
+          src: bundledFunctionPath,
+          path: '.netlify/functions/bundled-function-1.zip',
+        })
+        .withContentFile({
+          path: '.netlify/functions/manifest.json',
+          content: JSON.stringify({
+            functions: [
+              {
+                ...bundledFunctionData,
+                path: path.join(builder.directory, '.netlify', 'functions', 'bundled-function-1.zip'),
+              },
+            ],
+            timestamp: Date.now() - age,
+            version: 1,
+          }),
+        })
+        .withContentFile({
+          path: 'out/index.html',
+          content: 'Hello world',
+        })
+        .withFunction({
+          path: 'bundled-function-1.js',
+          handler: async () => ({
+            statusCode: 200,
+            body: 'Bundled at deployment',
+          }),
+        })
+        .buildAsync()
 
-        t.is(await got(`${deployUrl}/.netlify/functions/bundled-function-1`).text(), 'Bundled at deployment')
-      })
-    },
-  )
+      const { deploy_url: deployUrl } = await callCli(
+        ['deploy', '--json'],
+        {
+          cwd: builder.directory,
+          env: { NETLIFY_SITE_ID: context.siteId },
+        },
+        true,
+      )
 
-  test.after('cleanup', async (t) => {
-    const { siteId } = t.context
-    console.log(`deleting test site "${SITE_NAME}". ${siteId}`)
-    await callCli(['sites:delete', siteId, '--force'])
+      const response = await fetch(`${deployUrl}/.netlify/functions/bundled-function-1`).then((res) => res.text())
+      t.expect(response).toEqual('Bundled at deployment')
+    })
   })
-}
+})
 
 test('always pass, used for forked PRs since ava fails when no tests are present', (t) => {
```

Comment:
do we still need that test?

Reply:
Not sure. 
Should I remove it from the file?